app/src/main/java/it/ministerodellasalute/immuni/ui/dialog/BottomSheetDialogLightFragment.kt

```diff
@@ -80,7 +80,7 @@ abstract class BottomSheetDialogLightFragment : BottomSheetDialogFragment() {
     // light theme
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR should be set for API M+ too. So probably it is needed some additional if block.